pom.xml

```diff
@@ -15,7 +15,7 @@
   <!-- Artifact Configuration -->
   <groupId>org.jboss.arquillian.container</groupId>
   <artifactId>arquillian-parent-glassfish6</artifactId>
-  <version>1.0.0.Final-SNAPSHOT</version>
+  <version>6.0.0.Final-SNAPSHOT</version>
```

Comment:
What's the idea behind the versioning scheme? Is `X` (from `X.Y.Z.Final-SNAPSHOT`) a hint about the targeted GF version?

Reply:
Yes, that is correct. Since this container will be used with many versions of GlassFish, we need a way to differentiate which version is targeted.